src/fragments/lib/datastore/ios/data-access/query-predicate-or-snippet.mdx

```diff
@@ -7,7 +7,7 @@ let p = Post.keys
 do {
     let result = try await Amplify.DataStore.query(
         Post.self,
-        where: p.rating == nil || p.status == PostStatus.draft
+        where: p.rating == nil || p.status == PostStatus.active
```

Comment:
same here. in this case, shouldn't this be `.published`?

Reply:
the schema i used from the first page of the docs is 

```
type Post @model {
  id: ID!
  title: String!
  status: PostStatus!
  rating: Int
  content: String
}

enum PostStatus {
  ACTIVE
  INACTIVE
}
```
is there another PostStatus somewhere that uses .published ?

Reply:
my mistake, you're correct..I was using a different schema...